obk_deploy

```diff
@@ -147,7 +147,7 @@ def deploy():
             cmd.append("env.{}={}".format(key[9:], value))
         if key.startswith("OBK_SECRET_ODOO_"):
             cmd.append("--set")
-            cmd.append("env.{}={}".format(key[16:], value))
+            cmd.append("secrets.env.{}={}".format(key[16:], value))
```

Comment:
`secret_env` instead of `secrets.env` so values are flatter (and somehow env, and secret env are at the same level)?

Reply:
51f0113aef94e086eade3f64864af14e06f3c765